bindings/trycp_runner/src/common.rs

```diff
@@ -314,7 +314,7 @@ where
         let agent_name = agent_name.clone();
         async move {
             let logs = client
-                .download_logs(agent_name, Some(Duration::from_secs(180)))
+                .download_logs(agent_name, Some(Duration::from_secs(10 * 60)))
```

Comment:
When the logs are outputting a lot of information it can take a few minutes to get everything